File: depense/implementation/src/main/java/org/cocktail/gfc/depense/metier/modele/codeanalytique/CodeAnalytiqueRepository.java
package org.cocktail.gfc.depense.metier.modele.codeanalytique;

import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
import org.springframework.data.repository.CrudRepository;
import org.springframework.data.repository.PagingAndSortingRepository;

public interface CodeAnalytiqueRepository extends PagingAndSortingRepository<CodeAnalytique, Long>, JpaSpecificationExecutor<CodeAnalytique> {

}

File: depense/depense-core/depense-core-implementation/src/main/java/org/cocktail/gfc/depense/metier/modele/codeanalytique/CodeAnalytiqueRepository.kt
package org.cocktail.gfc.depense.metier.modele.codeanalytique

import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.data.repository.PagingAndSortingRepository

interface CodeAnalytiqueRepository : PagingAndSortingRepository<CodeAnalytique, Long>, JpaSpecificationExecutor<CodeAnalytique>